src/get-started/create-service.md

```diff
@@ -890,17 +926,17 @@ This request returns the transaction hash:
 
```

Comment:
In the skipped fragment above:

> This transaction transfers 15 coins from the first wallet to the second.

15 coins should be replaced with 5. Also, I'd suggest to use "token" instead of "coin" everywhere in the article.